lib/t/calls.ex

```diff
@@ -177,20 +222,22 @@ defmodule T.Calls do
   def accept_call(call_id, now \\ utc_now()) do
     Multi.new()
     |> set_call_accepted_at(call_id, now)
-    |> maybe_get_match_id()
-    |> maybe_insert_call_start_event()
+    |> get_match_id()
+    |> maybe_insert_call_start_event(now)
+    # TODO overwrite prev interaction (call attempt)?
```

Comment:
yes to this, and also `call_end` should probably overwrite all of these and include duration or (time start + time end) into `interaction` content

Reply:
https://github.com/getsince/test3/pull/540/commits/433a640bda6f50af544e04314670d439223aa14b